### PR TITLE
fix team alignments

### LINKED ISSUE #3091: Random Team alignments is missing borders
**Describe the bug**
Missing borders for team alignments window and some lines that shouldn't there:
grafik

**Did it work earlier?**
Same problem in 1.3.2

**Version**
 - OS: Ubuntu
 - Version: nightly

### PR DESCRIPTION
**Pull request by Laserlicht**
fixes #3091

image

### CODE REVIEW

File: client/lobby/RandomMapTab.cpp [hunk 1]
```diff
@@ -107,7 +107,7 @@ RandomMapTab::RandomMapTab():
 	//new callbacks available only from mod
 	addCallback("teamAlignments", [&](int)
 	{
-		GH.windows().createAndPushWindow<TeamAlignmentsWidget>(*this);
+		GH.windows().createAndPushWindow<TeamAlignments>(*this);
```
Comment (IvanSavenko) [inline client/lobby/RandomMapTab.cpp:110]: Why we now need two windows for this?
TeamAlignmentsWidget and TeamAlignments?
What's the difference between them?

Comment (Laserlicht) [reply client/lobby/RandomMapTab.cpp:110]: One is the Window, the other a Widget inside of this Window. I don't find a nice way to add border without it.

**Comment by Nordsoft91** [comment]
That's something really weird.

There is configurable widget, which is described in json file.
Instead of replacing regular texture with textureplayercolored in json, we are going to have **new object** which is basically adding hardcoded texture and then on the top of it creating old configurable object?

**Comment by Laserlicht** [comment]
> That's something really weird.
> 
> There is configurable widget, which is described in json file. Instead of replacing regular texture with textureplayercolored in json, we are going to have **new object** which is basically adding hardcoded texture and then on the top of it creating old configurable object?

Yes, it's really weird... There is no `config/widgets/randomMapTeamsWidget.json`...
I don't know why it works without json...

**Comment by Nordsoft91** [comment]
@Laserlicht , this file is located in extras. When extras disabled you will see classic ui, not hdmod ui

**Comment by Laserlicht** [comment]
@Nordsoft91 Use the config now.

Adjustment to mod: https://github.com/vcmi-mods/vcmi-extras/pull/19

**Review by Nordsoft91 — APPROVED** [review]
1) Ideally, would be good to have frame as configurable widget too, however not sure how many use cases could be
2) cancel button is covered by frame if only 2-3 players are selected, would be good to fix, but not regression

**Pull request merged**